task/golangci-lint/0.1/README.md

```diff
@@ -0,0 +1,52 @@
+# Unit Tests for Go Code
+
+The golangci-lint Task runs a Golang linter.
+The task is tailored to be triggered by Prow as a ProwJob.
+It uses a golangci-lint tool.
+The GolangCI tool config is stored in the `.golangci.yaml` file in a repository hosting tested code.
+The task does not accept additional parameters to configure the linter. By default, it runs linter against all modules
+found under task working directory.
+
+## Compatibility
+
+- Tekton v0.36.0 and above
+
+## Install
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/kyma-project/test-infra/main/task/golangci-lint/0.1/golangci-lint.yaml
+```
```

Comment:
URL return 404 for me :/

Reply:
Because this file is added in this PR. It doesn't exist under this path until we merge PR.